src/openrct2-ui/input/ShortcutInput.cpp

```diff
@@ -247,6 +247,19 @@ std::string_view ShortcutInput::GetLocalisedKeyName(uint32_t key)
         { SDLK_KP_0, STR_SHORTCUT_NUMPAD_0 },
         { SDLK_KP_PERIOD, STR_SHORTCUT_NUMPAD_PERIOD },
         { SDLK_CAPSLOCK, STR_SHORTCUT_NUMPAD_PERIOD },
```

Comment:
Why was `SDLK_CAPSLOCK` mapped to `STR_SHORTCUT_NUMPAD_PERIOD`? This seems wrong.

I just noticed because it's the only key repeated

Reply:
That looks like a mistake alright. Nice catch!

Reply:
@Garvity would you mind doing another commit fixing this oversight? I filed #24448 to track it.

Also, add changelog entries for each on https://github.com/OpenRCT2/OpenRCT2/blob/develop/distribution/changelog.txt